public/editor/scripts/editor/js/fc/publisher.js

```diff
@@ -273,8 +273,8 @@ define(function(require) {
 
     // Expose the published state with the updated link
     published.link
-      .attr("href", publishUrl)
-      .text(publishUrl);
+      .attr("href", "https://" + publishUrl)
+      .text("https://" + publishUrl);
```

Comment:
Two questions:

* Why does `publishUrl` not already have this?  It seems to me like the right fix is to add it to that string, vs. appending here, which feels brittle, especially if we ever do add it in the future.
* If you *do* fix it this way, I think it's wrong to force `https://` on dev, since it will be running on `http://`.  Why not use the value of `window.location.protocol` to match what we're loading the app with at runtime?